evaporate.js

```diff
@@ -597,7 +597,6 @@
                         setStatus(COMPLETE);
                     } else {
                         l.d('headObject not found on S3.');
-                        me.name = awsKey;
```

Comment:
I agree. How about we merge everything but that and spend a bit more time making sure removing it is correct?